test/alternator/test_returnvalues.py

```diff
@@ -83,6 +83,28 @@ def test_put_item_returnvalues(test_table_s):
     with pytest.raises(ClientError, match='ValidationException'):
         test_table_s.put_item(Item={'p': p, 'a': 'hello'}, ReturnValues='none')
 
+# Testing feature which returns values only on delete condition failures.
```

Comment:
Please explicitly use the feature's name, ReturnValuesOnConditionCheckFailure in the comment.  Same in all copies below. And please make the comments fit the test (here you said "delete" but it's a "put") and a bit clearer (it's not a "delete condition failure", it's a "DeleteItem with a failed condition expression").

Reply:
ok